.envrc

```diff
@@ -0,0 +1,4 @@
+export PATH=$PWD/vendor/bin:$PATH
+export PATH=$PWD/.tools:$PATH
```

Comment:
curious, what does this allow you to do? what does it simplify exactly? (keeping in mind that we have `make *` commands and `docker-compose.yaml`

Reply:
you could type `php-cs-fixer` and it will use the one from `/path/to/infection/.tools/php-cs-fixer`.

>keeping in mind that we have make * commands and docker-compose.yaml

Yes, it's not really intended as a replacement, more of if for whatever reason you wanna use that executable file it will be slightly easier. I hesitated to add it but I'm adding the Composer bins already, I thought adding this would make sense too.

We could totally do without any of this tbh, the only part I really am keen to have is the `.envrc.local` part